frog/imports/ui/LandingPage/index.js

```diff
@@ -137,39 +148,52 @@ const LandingPage = ({ history }) => {
           complete control over the progress of the class and all it takes is
           three steps.
         </Typography>
-        <Button
-          disableFocusRipple
-          disableRipple
-          className={classes.lineButton}
-          onClick={() => history.push('/wizard')}
-        >
-          Try it out now
-        </Button>
+        <div className={classes.alignCenterDiv}>
+          <Button
```

Comment:
Consider using the FROG Button

Reply:
Just tried to replace it. However `<Button>` is used at other places with some custom styles and replacing it causes problems. I do agree though, we should be using the FROG button as a standard. I'll try to replace all the buttons on the page soon when redesigning. 
Ps. Suggestion: I find that using the FROG button causes a line break. I suggest adding a display: inline-block.